Library/Homebrew/cask/audit.rb

```diff
@@ -545,6 +545,38 @@ def check_livecheck_version
       false
     end
 
+    def check_livecheck_min_os
+      return unless online?
+      return unless cask.livecheckable?
+      return unless cask.livecheck.strategy == :sparkle
+
+      out, _, status = curl_output(cask.livecheck.url)
+      return unless status.success?
+
+      require "rexml/document"
+
+      xml = begin
+        REXML::Document.new(out)
+      rescue REXML::ParseException
+        nil
+      end
+
+      return if xml.blank?
+
+      item = xml.get_elements("//rss//channel//item").first
+      return if item.blank?
+
+      min_os = item.elements["sparkle:minimumSystemVersion"].text
```

Comment:
```suggestion
      min_os = item.elements["sparkle:minimumSystemVersion"]&.text
```

If there's no minimum version, then this will be nil.